src/scripting/lua_bases.cc

```diff
@@ -955,14 +974,13 @@ int LuaPlayerBase::place_ship(lua_State* L) {
 /* RST
    .. method:: conquer(f[, radius=1])
 
-      Conquer this area around the given field if it does not belong to the
+      Conquer an area around the given field if it does not belong to the
       player already. This will conquer the fields no matter who owns it at the
       moment.
 
-      :arg f: center field for conquering
-      :type f: :class:`wl.map.Field`
-      :arg radius: radius to conquer around. Default value makes this call
-         conquer 7 fields
+      :arg f: Center field for conquering.
+      :type f: :class:`~wl.map.Field`
+      :arg radius: (Optional) Radius to conquer around. Defaults to conquer 7 fields.
```

Comment:
7 fields is misleading as the sentence before is talking about a radius. To the casual reader this sounds like the default is a radius of 7 fields. Perhaps rephrase "Defaults to conquer a radius of 1 (7 fields in total)"?